pkg/arguments/interactive.go

```diff
@@ -280,12 +281,35 @@ func PromptFilePath(fs *pflag.FlagSet, flagName string, required bool) error {
 			if err != nil {
 				return err
 			}
+			response, err = resolveRelativePath(response)
+			if err != nil {
+				return err
+			}
 			fs.Set(flagName, response)
 		}
 		return nil
 	})
 }
 
+// Golang does not support tilde file paths https://github.com/golang/go/issues/57569
+// However, we try to resolve this by manually so user can proceed further
+func resolveRelativePath(path string) (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+	dir := usr.HomeDir
+	if path == "~" {
+		// In case of "~", "else if" would not catch that
+		path = dir
+	} else if strings.HasPrefix(path, "~/") {
+		// Use strings.HasPrefix so we don't match paths like
+		// "/something/~/something/"
+		path = filepath.Join(dir, path[2:])
+	}
```

Comment:
can we simplify this logic?

Reply:
Removed the earlier if condition and added assignment to a new variable